### PR TITLE
Features/improve tests

### PR DESCRIPTION
**Pull request by Szarad**

### CODE REVIEW

**Comment by Szarad** [comment]
Test should be run as usual using: `$pytest -vv`
Updates should be done: `$pytest --snapshot-update` (only run when changing input for test!)

File: Pipfile [hunk 1]
```diff
@@ -19,6 +19,7 @@ opencensus-ext-azure = "*"
 redis = "*"
 platformshconfig = "*"
 fakeredis = "*"
+pytest-snapshot = "*"
```
Comment (lsmith77) [inline Pipfile]: this should probably be in the dev section?

**Review by lsmith77 — CHANGES_REQUESTED** [review]
amazing!!! this will save soooo much time.

can you look at moving the dep to --dev?

and you can merge

File: Pipfile (hunk 1 above)
Comment (Szarad) [reply Pipfile]: ah thank you, I will change it.

**Review by lsmith77 — APPROVED** [review]

**Pull request merged**